### ISSUE TITLE
Tableau ssl_verify does not work

### ISSUE THREAD

**Issue by dvenza** [datahub-project/datahub#6709]
**Describe the bug**
The ssl_verify option for tableau injestion does not work

**To Reproduce**
Set the option to true and try to connect to a tableau server with a custom certificate.

**Expected behavior**
Connection is succesful.

**Additional context**

This code (from `tableau.py`) establishes the connection before setting the option:
```
server = Server(self.connect_uri, use_server_version=True)

# From https://stackoverflow.com/a/50159273/5004662.
server._session.verify = self.ssl_verify
server._session.trust_env = False
```

With the following the connection is succesful:
```
server = Server(self.connect_uri, use_server_version=True, http_options={'verify': self.ssl_verify})

# From https://stackoverflow.com/a/50159273/5004662.
# server._session.verify = self.ssl_verify
# server._session.trust_env = False
```

**Comment by mayurinehate** [comment]
Hey @dvenza this is recently fixed by this [PR](https://github.com/datahub-project/datahub/pull/6682) and is available in 0.9.3.3rc3 if you want to try it out.

**Comment by dvenza** [comment]
Perfect, thanks. The issue can be closed.
